### PR TITLE
Abe644

### PR DESCRIPTION
**Pull request by michaelleahy**
@lookinginstereo @joshhaynes @jsatria 

Passthrough.  Plus client code that's not yet merged.  I'll write the RCU side tonight.

### CODE REVIEW

File: adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+  def deleteCharity(charityId: String): EssentialAction = Action.async {
+    client.deleteCharity(charityId)
+  }
+
+  def getAllCharities: EssentialAction = Action.async {
+    client.getAllCharities
+  }
+
+  def getCharity(charityId: String): EssentialAction = Action.async {
+
```
Comment (joshhaynes) [inline adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala]: Whitespace

File: adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+    future: Future[Unit]): Future[Result] =
+    future.map(_ => NoContent) recover defaultRecoverBlock
+
+  /**
+   * Overrides the recovery block that is built in to [[BaseController]]
+   * to handle our specific exceptions.
+   */
+  private def handleResponseException(response: WSResponse): Result = {
+    response.status match {
+      case 401 => Unauthorized(response.body)
```
Comment (joshhaynes) [inline adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala]: This isn't wrong, but it strikes me this should be handled somewhere else -- this seems like boilerplate

**Comment by joshhaynes** [comment]
LGTM

File: models/src/main/scala/com/rallyhealth/donations/model/ImitationDonationsDataHelper.scala [hunk 3]
```diff
@@ -35,7 +34,7 @@ trait ImitationDonationsDataHelper {
     CharityInfo(
       id = "pave",
       templateName = "Pave the Bay Template Name",
-      logoUrl = "http://www.mthai.com/en/wp-content/uploads/2015/01/its-mine_quickmeme-com.jpg",
+      logoUrl = "http://farm4.static.flickr.com/3138/3041315782_c6eff35c43_m.jpg",
```
Comment (lookinginstereo) [inline models/src/main/scala/com/rallyhealth/donations/model/ImitationDonationsDataHelper.scala]: :+1:

File: adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala (hunk 2 above)
Comment (michaelleahy) [inline adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala]: If definitely is.  The problem is that deciding how to handle it is a bit nebulous.  Any time we try to prescribe around here, it ends up cornering someone.

File: adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala (hunk 1 above)
Comment (michaelleahy) [inline adminWeb/app/com/rallyhealth/donations/controller/AdminPassthroughController.scala]: Lousy auto formatter not fixing that...

File: models/src/main/scala/com/rallyhealth/donations/model/ImitationDonationsDataHelper.scala (hunk 3 above)
Comment (michaelleahy) [inline models/src/main/scala/com/rallyhealth/donations/model/ImitationDonationsDataHelper.scala]: :bow:

**Comment by lookinginstereo** [comment]
LGTM

**Pull request merged**